### ISSUE TITLE
Add fig-alt text to all figure code chunks

### ISSUE THREAD

**Issue by ianperaltahirujo** [ianperaltahirujo/Intro-to-R#2]
All figure chunks in the QMD are missing fig-alt options, which are required for accessibility and for meeting the Adept rubric standard.
